solr/solrj/src/java/org/apache/solr/client/solrj/impl/LBSolrClient.java

```diff
@@ -155,6 +159,7 @@ public ServerIterator(Req req, Map<String, ServerWrapper> zombieServers) {
       this.req = req;
       this.zombieServers = zombieServers;
       this.timeAllowedNano = getTimeAllowedInNanos(req.getRequest());
+      log.info("TimeAllowedNano:{}", this.timeAllowedNano);
```

Comment:
Are you sure we should log at info level here?  This seems more like a debug situation.

Reply:
Thank you, I totally forget that on creating this PR.